core/templates/base-components/base-content.component.html

```diff
@@ -49,7 +49,7 @@
                                           [subheaderText]="getSubheaderText()">
                 </oppia-top-navigation-bar>
                 <div class="collapse d-lg-inline-flex navbar-collapse oppia-navbar-collapse ng-cloak">
-                  <div [ngClass]="{'mr-auto': !isLanguageRTL(), 'ml-auto': isLanguageRTL()}">
+                  <div class="hide-nav-heading" [ngClass]="{'mr-auto': !isLanguageRTL(), 'ml-auto': isLanguageRTL()}">
```

Comment:
Please rename the class to `oppia-hide-nav-heading` (we generally prefix class names with 'oppia-').

Reply:
Hi @Lawful2002,
Class name is updated to `oppia-hide-nav-heading` 
`python -m scripts.rtl_css --mode generate` is not working because RTL CSS is moved to webpack. PTAL. Thanks!

![screenshot1](https://user-images.githubusercontent.com/86031203/178957795-178e0e02-5153-4054-b638-5d35736f77e9.jpeg)

![screenshot](https://user-images.githubusercontent.com/86031203/178957080-d51762a7-eb8f-45d9-b5dc-6b15602ddcad.jpeg)